lib/node.js

```diff
@@ -2,12 +2,23 @@ var request = require("request");
 
 module.exports = getJSON;
 
-function getJSON (url, callback) {
-  request({ url: url, json: true }, function (error, response, body) {
+function getJSON (url, options, callback) {
+  if (!callback) {
+    callback = options;
+    options  = null;
+  }
+
+  var defaultOptions = { url: url, json: true };
+
+  if (options) {
+    defaultOptions = Object.assign(defaultOptions, options);
```

Comment:
`Object.assign` is only available since node 4. I'm fine with this, but please add a stanza to package.json to make node support explicit:

```json
{
  "engines": {
    "node": ">=4"
  }
}
```